### PR TITLE
task: stratify save for reuse

### PR DESCRIPTION
**Pull request by asylves1**
# Description

Screenshot 2024-09-03 at 9 27 10 AM
Resolves #(issue)

### CODE REVIEW

File: packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue [hunk 1]
```diff
@@ -77,6 +78,17 @@
 				v-model:output="selectedOutputId"
 				is-selectable
 			>
+				<section class="right-side">
+					<Button
+						label="Save for re-use"
+						size="small"
+						outlined
+						dis
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue:86]: `dis` ??

File: packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue [hunk 2]
```diff
@@ -442,6 +459,43 @@ const onSelection = (id: string) => {
...
+
+function updateNodeOutput(model: Model) {
+	if (!selectedOutputId.value || !model) return;
+
+	amr.value = model;
+	const outputPort = _.cloneDeep(props.node.outputs?.find((port) => port.id === selectedOutputId.value));
+
+	if (!outputPort) return;
+	outputPort.label = model.header.name;
+	outputPort.value = [model.id];
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue:479]: I presume the model.id hasn't changed? If so I wouldn't bother updating this field.

Comment (asylves1) [reply packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue]: removed

**Review by mwdchang — APPROVED** [review]
thanks

**Review by YohannParis — APPROVED** [review]

**Pull request merged**